vcs_requirements.txt

```diff
@@ -1,6 +1,6 @@
 # version control systems requirements
 guacamol_baselines @ git+https://github.com/GT4SD/guacamol_baselines.git@v0.0.2
-molecule_generation @ git+https://github.com/microsoft/molecule-generation.git@v0.2.0
+molecule_generation @ git+https://github.com/GT4SD/molecule-generation.git@main
```

Comment:
cool, if there were no major changes, we could even open a PR on the original repo to suggest the update

Reply:
Doing that now!